databases/Assignment2_answers.sql

```diff
@@ -0,0 +1,316 @@
+1. Which laptop models are shown in the Product relation?
+SELECT DISTINCT(Model)
+FROM Product
+WHERE Type = laptop;
```

Comment:
Depending on whether Model is the primary key, or if guaranteed unique, we might not need the `DISTINCT`. But it's not incorrect and good to think about